.editorconfig

```diff
@@ -1,3 +1,3 @@
 [*]
 indent_style = tab
-tab_width = 16
+tab_width = 8
```

Comment:
```suggestion
tab_width = 4
```

Reply:
If we’re gonna decrease it, we could just as well go all the way. #reclaimyourscreenspace

```suggestion
tab_width = 0
```

Reply:
Y'all arguing over a number when there's no less than 14 bytes to be saved here.

```suggestion
```